dev-haskell/binary/binary-0.8.5.1.ebuild

```diff
@@ -15,7 +15,7 @@ SRC_URI="mirror://hackage/packages/archive/${PN}/${PV}/${P}.tar.gz"
 LICENSE="BSD"
 SLOT="0/${PV}"
 # keep in sync with ghc-8.2.2 and ghc-8.4.2
-# KEYWORDS="~alpha ~amd64 ~x86 ~amd64-linux ~x86-linux"
+KEYWORDS="~alpha ~amd64 ~x86 ~amd64-linux ~x86-linux"
```

Comment:
You need to keep KEYWORDS in sync with ghc-8.4.3 and drop ~alpha. Other wise alpha users will get two `binary` package versions and we will get diamond dependency problem.

It is only a problem for some packages bundled with ghc and exposed from ghc ebuilds. I'll try to write a doc on it if we don't already have it.